show/muxcable.py

```diff
@@ -155,22 +156,25 @@ def status(port, json_output):
 
         muxcable_info_dict[asic_index] = per_npu_statedb[asic_index].get_all(
             per_npu_statedb[asic_index].STATE_DB, 'MUX_CABLE_TABLE|{}'.format(port))
+        muxcable_health_dict[asic_index] = per_npu_statedb[asic_index].get_all(
+            per_npu_statedb[asic_index].STATE_DB, 'MUX_LINKMGR_TABLE|{}'.format(port))
         if muxcable_info_dict[asic_index] is not None:
             logical_key = "MUX_CABLE_TABLE"+"|"+port
+            logical_health_key = "MUX_LINKMGR_TABLE"+"|"+port
```

Comment:
LGTM alerts that `logical_health_key` is unused.
Also, please add spaces around `+` signs.
Furthermore, suggest `logical_key = "MUX_CABLE_TABLE|" + port` or `logical_key = "MUX_CABLE_TABLE|{}".format(port)`

Reply:
fixed